samples/image_viewer.cpp

```diff
@@ -246,7 +246,7 @@ static void loadImage(App& app, Engine* engine, const Path& filename) {
             .height(h)
             .levels(0xff)
             .format(channels == 3 ?
-                    Texture::InternalFormat::RGB16F : Texture::InternalFormat::RGBA16F)
+                    Texture::InternalFormat::RGB16F : Texture::InternalFormat::RGBA32F)
```

Comment:
I don't understand this change. It seems like this might be an issue with a particular backend? Why is RGBA16F not working?

Reply:
Definitely seems like a backend issue. Actually I would expect RGB16F to be the issue, not RGBA16F.